test/sql/test_ddl/T/create_table_like

```diff
@@ -0,0 +1,35 @@
+-- name: test_ddl_create_table_like
+create database db_${uuid0};
+use db_${uuid0};
+CREATE TABLE t1 (id1 int, id2 int, id3 double) DUPLICATE KEY(id1) COMMENT "c1" DISTRIBUTED BY HASH(id1) BUCKETS 1 properties('replication_num' = '1');
+CREATE TABLE t2 LIKE t1;
+SHOW CREATE TABLE t2;
+
+CREATE TABLE `test_pk_tbl1` (
+`k1` DATE,
+`k2` DATETIME,
+`k3` VARCHAR(20),
+`k4` VARCHAR(20),
+`k5` BOOLEAN,
+`v1` TINYINT NULL,
+`v2` SMALLINT NULL,
+`v3` INT NULL,
+`v4` BIGINT NOT NULL,
+INDEX init_bitmap_index (k2) USING BITMAP
```

Comment:
why test the olap table?

Reply:
UT covered Hive table and thought it would be nice to have the olap table covered as well